Templates/framework/tvOS/Template.framework/Headers/README

```diff
@@ -0,0 +1 @@
+Placeholder for empty git directory.
```

Comment:
Is the structure of a tvOS framework different than iOS? If not, can we use the same framework template for both?

Reply:
Done!